packages/client/components/GroupingKanbanColumn.tsx

```diff
@@ -85,6 +86,8 @@ interface Props {
   swipeColumn?: SwipeColumn
 }
 
+const AUTO_GROUPING_THRESHOLD = 0.5
```

Comment:
0.5 feels a bit to high for an initial coefficient, is that what we had it as before? getGroupMatrix will increment this up to 5 times in order to find a good fit, no harm in starting low